lib/grizzly/zwave/command_classes/thermostat_fan_mode.ex

```diff
@@ -22,12 +33,28 @@ defmodule Grizzly.ZWave.CommandClasses.ThermostatFanMode do
   def encode_mode(:low), do: 0x01
   def encode_mode(:auto_high), do: 0x02
   def encode_mode(:high), do: 0x03
+  def encode_mode(:auto_medium), do: 0x04
+  def encode_mode(:medium), do: 0x05
+  def encode_mode(:circulation), do: 0x06
+  def encode_mode(:humidity_circulation), do: 0x07
+  def encode_mode(:left_right), do: 0x08
+  def encode_mode(:up_down), do: 0x09
+  def encode_mode(:quiet), do: 0x0A
+  def encode_mode(:external_circulation), do: 0x0B
 
   @spec decode_mode(byte) :: {:ok, mode} | {:error, DecodeError.t()}
   def decode_mode(0x00), do: {:ok, :auto_low}
   def decode_mode(0x01), do: {:ok, :low}
   def decode_mode(0x02), do: {:ok, :auto_high}
   def decode_mode(0x03), do: {:ok, :high}
+  def decode_mode(0x04), do: {:ok, :auto_medium}
+  def decode_mode(0x05), do: {:ok, :medium}
+  def decode_mode(0x06), do: {:ok, :circulation}
+  def decode_mode(0x07), do: {:ok, :humidity_circulation}
+  def decode_mode(0x08), do: {:ok, :left_right}
+  def decode_mode(0x09), do: {:ok, :up_down}
+  def decode_mode(0x0A), do: {:ok, :quiet}
+  def decode_mode(0x0B), do: {:ok, :external_circulation}
```

Comment:
Any reason not to have a `decode_mode(_), do: {:ok, :unknown}` or something similar? 

I.e., don't crash on unknown user input? Or at least, I'm assuming this worked before since Jon had his thermostat set on one of the known modes.

Reply:
I see comments on slack that this is another issue... Don't worry about this then....

Reply:
Yeah, we can do that. My hesitation is if we just do a one-off fix for this that is temporary the codebase will become inconsistent and harder to manage until there's time to provide a better solution.

I have this issue open: https://github.com/smartrent/grizzly/issues/304 to help remind me this needs to get done.

Reply:
Just to be clear, I agree and I think that is going to be the solution, but I want to just make a pass at implementing it on the whole codebase in one PR.

I think it is easy just maybe a little tedious, but find and replace should help.